src/log.rs

```diff
@@ -0,0 +1,57 @@
+use std::fs::File;
+
+use simplelog::{Config, LevelFilter, SimpleLogger, WriteLogger};
+
+#[used]
+#[cfg_attr(target_os = "linux", link_section = ".init_array")]
+pub static INITIALIZE: extern "C" fn() = kryoptic_log_init;
+
+/// Initializes a simple logger for tracing purposes based on the values of
+/// the environment variable KRYOPTIC_TRACE:
+/// - stdout -> logs to standard output
```

Comment:
one more thing, should we support `stderr`? I would find it more common than stdout, which should have the application output, rather than the tracing information.

Reply:
I can;t support stderr directly, but as you can see in the example above you can easily use KRYOPTIC_TRACE=/dev/stderr

I know it is not exactly teh same thing (as it may not work correctly with redirection), but simplelog emits only to stdout (which is why I corrected the text which previously claimed stderr).